src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/interactivemaps/InteractiveMapPageObject.java

```diff
@@ -166,9 +167,11 @@ public void clickZoomOutButton() {
     PageObjectLogging.log("clickZoomOutButton", "Map zoom out was clicked", true, driver);
   }
 
-  public void clickOnPin(Integer pinListPosition) {
-    waitForElementVisibleByElement(mapFrame);
-    driver.switchTo().frame(mapFrame);
+  public void clickOnPin(Integer pinListPosition, boolean ...noFrame) {
```

Comment:
Because if I'm going to 
maps.wikia-services.com/api/v1/render/77/?hideAttr=1&isEmbeddedInWikiaApp&uselang=en
There is no iframe and I don't want to switch there. I could overwrite this method and skip driver.switchTo() but I prefered to add this optional method argument.